presto-orc/src/main/java/io/prestosql/orc/metadata/OrcMetadataReader.java

```diff
@@ -282,8 +282,17 @@ else if (statistics.hasBinaryStatistics()) {
             minAverageValueBytes = 0;
         }
 
+        Long numberOfValues = statistics.getNumberOfValues();
```

Comment:
The `numberOfValues` variable never changes, so I would suggest either removing it entirely or calculating it at the very beginning of the method and replacing all `statistics.getNumberOfValues()` calls with it. Also, since you no longer use `null` as a value, you could use a primitive `long` type instead.